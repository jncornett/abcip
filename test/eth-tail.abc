d ( stack="eth:ip4:tcp" )
a ( eth:tail="|1 2 3 4 5 6|" )
