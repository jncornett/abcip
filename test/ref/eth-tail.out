IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [none], cksum 0xb3dc (correct), win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0001 0000 4006 5cbc 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0000 0000 0000 5000  ..............P.
	0x0030:  2000 b3dc 0000 0102 0304 0506            ............
